airflow/utils/trigger_rule.py

```diff
@@ -32,6 +32,7 @@ class TriggerRule:
     NONE_SKIPPED = 'none_skipped'
     DUMMY = 'dummy'
     ALWAYS = 'always'
+    NONE_FAILED_AND_AT_LEAST_ONE_SUCCEED = "none_failed_and_at_least_one_succeed"
```

Comment:
This is tooooo long. Are you sure you wanted to change the name of `none_failed_or_skipped` and not `none_failed` based on https://github.com/apache/airflow/issues/17012#issuecomment-883089001

Reply:
`This is tooooo long` okay , any proposition ?

I'm surprise by your second question , none_failed is not the same than none_failed_or_skipped, my MR is about renaming none_failed_or_skipped nothing else.

Reply:
`NONE_FAILED_MIN_ONE_SUCCESS` (`none_failed_min_one_success`) seems a contender

Reply:
I like `none_failed_min_one_success` :+1: 

none_failed_one_success could lead to ambiguity, example :  " if there 2 previous success, does this trigger_rule also work ? "